pkg/cmd/stack/deploy.go

```diff
@@ -428,7 +428,7 @@ func getServicesWithFailedProbes(ctx context.Context, stack *model.Stack, svcNam
 	return failedServices
 }
 
-func getFailedStatusError(ctx context.Context, stack *model.Stack, svcName string, client kubernetes.Interface, config *rest.Config) error {
+func getFailedStatusError(ctx context.Context, stack *model.Stack, svcName string, client kubernetes.Interface) error {
```

Comment:
maybe we could change this func name to something like `getErrorDueToRestartLimit` or something like that? Just a suggestion to add more context about what kind of error will be checking